finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeOrderResource.java

```diff
@@ -69,6 +70,10 @@ private Response validate(OrderRequest orderRequest) {
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "createOrder",
+               summary = "Create orders",
+               description = "This operation create orders " + 
+                   "by an OrderRequest and send to the Bar and Kitchen services.")
```

Comment:
Update the description to
"This operation creates orders by using an OrderRequest and sends them to the Bar and Kitchen services."

Reply:
updated